JSDemos/Demos/Button/Icons/description.md

```diff
@@ -7,7 +7,7 @@ DevExtreme ships with its own [icon library](/Documentation/Guide/Themes_and_Sty
 Set the **icon** value to the image file path or URI.    
 
 - **3rd-party icon fonts**    
-You can import a 3rd-party font library (this example uses <a href="https://fontawesome.com/icons?d=gallery" target="_blank">Font Awesome</a>). In such cases, set the **icon** property to a name that identifies the required glyph. Look up names in the imported library's documentation.
+You can import a 3rd-party font library (this example uses <a href="https://fontawesome.com/v5.15/icons?d=gallery" target="_blank">Font Awesome</a>). In such cases, set the **icon** property to a name that identifies the required glyph. Look up names in the imported library's documentation.
```

Comment:
The old link redirects to the latest version. The new link specifies the version explicitly.